pkg/resource/endpoint/custom_update_api.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Use this file to add custom implementation for any operation of intercept
+// the autogenerated code that trigger an update on an endpoint
+
+package endpoint
+
+import (
+	"context"
+	"errors"
+
+	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
+	"github.com/aws/aws-controllers-k8s/pkg/requeue"
```

Comment:
This needs to change to `github.com/aws-controllers-k8s/runtime/pkg/requeue`

Reply:
good catch!!